site/en/blog/angular-image-directive/index.md

```diff
@@ -0,0 +1,215 @@
+
+
+<figure>
+{% Img src="image/IypihH3o5cSpEMVp5i08dp69otp2/epoxZjv8snr3xdEKl9UB.png", alt="LCP for Angular websites over the last one year", width="800", height="711" %}
+</figure>
+
+
+Looking at the Core Web Vitals scores, the percentage of Angular origins that meet "good" LCP thresholds still needs work. Only 18.74% of Angular sites had good LCP on mobile in June 2022. Since images are the LCP element for more than 70% of the web pages on mobile and desktop, unoptimized LCP images could be one of the primary causes of poorer LCP on Angular websites. 
+
+The Angular image directive was designed to l help improve these numbers. 
+
+
+## MVP for the NgOptimizedImage directive
+
+The MVP of the Angular image directive builds on lessons from the image components Aurora has built to date while adapting the design to Angular’s client-side rendering experience. Many of the standard image optimization problems have been addressed by either:
+
+* Providing strong defaults or 
+* Throwing errors/warnings to ensure conformance to best practices.
+
+The highlights of the design are as follows:
+
+1. **Intelligent lazy loading**
+
+    Images that are invisible to the user on page load (e.g., below-the-fold images or hidden carousel images) should ideally be [lazy-loaded](https://web.dev/lazy-loading-images/). Lazy loading frees up browser resources to load other critical text, media, or scripts. Most images are non-critical and should be lazy-loaded, but only [7.8% of pages](https://almanac.httparchive.org/en/2021/resource-hints#fig-16) used native lazy loading in 2021.
+
+    The Angular image directive lazy loads non-critical images by default and only eagerly loads images specially marked as `priority`. This ensures that most images exhibit optimal loading behavior.
+
+
+2. **Prioritization of critical images**
+
+    Adding resource hints (e.g., <code>[preload](https://web.dev/preload-critical-assets/)</code> or <code>[preconnect](https://web.dev/uses-rel-preconnect/)</code>) to prioritize the loading of critical images is a [recommended best practice](https://web.dev/preload-critical-assets/). However, most apps are not using them. According to the 2021 Web Almanac, only [12.7% of mobile pages](https://almanac.httparchive.org/en/2021/resource-hints#fig-2) use preconnect hints and [only 22.1% of mobile pages](https://almanac.httparchive.org/en/2021/resource-hints#fig-2) use preload hints.
+
+
+	The image directive acts on two fronts when images are marked as priority.
+
+    * It sets the <code>[fetchpriority](https://web.dev/priority-hints/#increase-the-priority-of-the-lcp-image)</code> of the image to <code>"high"</code> so that the browser knows that it should download the image with a high priority.
```

Comment:
```suggestion
    * It sets the `[fetchpriority](https://web.dev/priority-hints/#increase-the-priority-of-the-lcp-image)` of the image to `"high"` so that the browser knows that it should download the image with a high priority.
```